### PR TITLE
New tmpfile

### PR DESCRIPTION
**Pull request by elbing**
Change-Id: I5149f34dad13fdd4dbd03571eb6dd52d1441c16f

### CODE REVIEW

File: lib/ap/plan9/tmpfile.c [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	n = create(name, 64|2, 0777); /* remove-on-close */
+	if(n==-1){
+		_syserrno();
+		return NULL;
+	}
+	_fdinfo[n].flags = FD_ISOPEN;
+	_fdinfo[n].oflags = 2;
+
+	f = fdopen(n, "w+");
+    if (!f) close(n);
```
Comment (sevki) [inline lib/ap/plan9/tmpfile.c]: indentation?

**Comment by elbing** [comment]
ha ha, ok, I'll review later. Style!

**Comment by elbing** [comment]
done

**Comment by elbing** [comment]
please, review this.

**Comment by elbing** [comment]
See #36

**Pull request closed without merging**